napari/_qt/layers/qt_image_layer.py

```diff
@@ -8,7 +9,8 @@ class QtImageControls(QtBaseImageControls):
     def __init__(self, layer):
         super().__init__(layer)
 
-        self.layer.events.interpolation.connect(self._on_interpolation_change)
```

Comment:
Eventually all these connection lines would get deleted and added to the controller